lib/tasks/publish_missed_scheduled_editions.rake

```diff
@@ -0,0 +1,11 @@
+namespace :editions do
+  desc "Publish editions missed in scheduled publishing"
+  task publish_missed_scheduled_editions: :environment do
+    scheduled_editions = Edition.scheduled_for_publishing.where(:publish_at.lt => Time.zone.now)
+
+    scheduled_editions.each do |edition|
+      ScheduledPublisher.new.perform(edition.id.to_s)
+      Rails.logger.info "Published edition with ID: #{edition.id}"
```

Comment:
Maybe we could be more explicit in the message about it being an edition that got missed from publication (to make searching for these issues in the logs easier).

We could also consider logging at `warn` rather than `info` level.